tests/test_adjoint_jacobian.py

```diff
@@ -797,15 +797,20 @@ def circuit(params):
         j_kokkos = qml.jacobian(qnode_kokkos)(params)
 
 
-def test_fail_adjoint_Hamiltonian():
+def test_adjoint_Hamiltonian():
     """Integration tests that compare to default.qubit for a large circuit containing parametrized
     operations and when using custom wire labels"""
 
     coeffs = [0.5, -0.25, 1.0]
-    obs = [qml.PauliX(0) @ qml.PauliX(1), qml.PauliZ(0) @ qml.PauliY(2), qml.PauliZ(3)]
+    obs = [
+        qml.PauliX(custom_wires[0]) @ qml.PauliX(custom_wires[1]),
+        qml.PauliZ(custom_wires[0]) @ qml.PauliY(custom_wires[2]),
+        qml.PauliZ(custom_wires[3]),
+    ]
     hamiltonian = qml.Hamiltonian(coeffs, obs)
 
     dev_kokkos = qml.device("lightning.kokkos", wires=custom_wires)
+    dev_cpu = qml.device("default.qubit", wires=custom_wires)
```

Comment:
Instead of `_cpu`, I suggest something like `_def` for the `default.qubit`.
```suggestion
    dev_def = qml.device("default.qubit", wires=custom_wires)
```

Reply:
Good idea!